### PR TITLE
Add unison.2.51.3

### PR DESCRIPTION
**Pull request by kit-ty-kate**
cc @gdt

### CODE REVIEW

File: packages/unison/unison.2.51.3/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+authors: [
+  "Jürgen Hötzel <juergen@hoetzel.info>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "mailto:unison-users@yahoogroups.com "
+dev-repo: "git://github.com/bcpierce00/unison.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
```
Comment (mseri) [inline packages/unison/unison.2.51.3/opam:12]: It seems to require a lower bound on ocaml:

```
#=== ERROR while compiling unison.2.51.3 ======================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/bcpierce00/unison/archive/v2.51.3.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/unison.2.51.3
# command              ~/.opam/4.02/bin/dune build -p unison -j 47
# exit-code            1
# env-file             ~/.opam/log/unison-15-34169e.env
# output-file          ~/.opam/log/unison-15-34169e.out
### output ###
#       ocamlc src/lwt/.lwt_lib.objs/byte/lwt.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.02/bin/ocamlc.opt -w -40 -warn-error -3-27-39 -no-strict-sequence -g -bin-annot -I src/lwt/.lwt_lib.objs/byte -no-alias-deps -o src/lwt/.lwt_lib.objs/byte/lwt.cmi -c -intf src/lwt/lwt.mli)
# /home/opam/.opam/4.02/bin/ocamlc.opt: unknown option '-no-strict-sequence'. 
```

File: packages/unison/unison.2.51.3/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "dune" {>= "1.3"}
+  "lablgtk"
```
Comment (mseri) [inline packages/unison/unison.2.51.3/opam:14]: I am not sure if the other failures depend on this, but from the various errors I think we may need some bounds for lablgtk as well

**Comment by gdt** [comment]
I am not up to speed on opam details, so I don't immediately follow if there is something that you want me to explain or comment on.   I certainly do not have any issues with unison being added to another packaging system.

2.51.4 is coming, but that should be an easy upgrade for packaging and I don't recommend waiting.

**Comment by gdt** [comment]
Also, unison in general does not depend on glibc; it works fine on NetBSD and macOS.  But there could be an "if linux then glibc" assumption, in which case a bug report to unison seems in order.

**Comment by kit-ty-kate** [comment]
I sent a PR in https://github.com/bcpierce00/unison/pull/485

**Comment by mseri** [comment]
Should we close this in favour of a new release with alpine support, after the PR is merged? Otherwise, it seems that this could be merged in the meantime (if I looked correctly, the CI build failures are just overzealous regexps picking up a warning).

**Pull request closed without merging**